### PR TITLE
Added build script, build info and Dockerfile for abyss

### PR DESCRIPTION
**Pull request by muskaan2199**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by nikhil-kalbande** [comment]
@MuskaanSheik - Travis build is failing, can you please retrigger build with some commit or post container logs for successful build

File: a/abyss/abyss_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+tar jxf boost_1_56_0.tar.bz2
+
+./autogen.sh
+
+mkdir build
+cd build/
+../configure --prefix=/usr/local/abyss
+make 
+make install
+make check
```
Comment (nikhil-kalbande) [inline a/abyss/abyss_ubi_8.5.sh:54]: @muskaan2199  Can you please add a code to handle build and test failures by returning exit code 1 and 2.
Rest looks good to me.

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: a/abyss/abyss_ubi_8.5.sh (hunk 1 above)
Comment (muskaan2199) [reply a/abyss/abyss_ubi_8.5.sh]: @nikhil-kalbande  Made the required changes

**Comment by seth-priya** [comment]
@nikhil-kalbande please review

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**